SFR/outer_fuel_pincell/make_SFR_pincell_model.py

```diff
@@ -0,0 +1,113 @@
+import openmc
+import numpy as np
+from argparse import ArgumentParser
+import openmc.material
+from materials import *
+import common_input as geom
+
+
+def settings(shannon_entropy: bool, height: float):
+    pitch = geom.pitch
+    lower_left = (-pitch / 2, -pitch / 2, 0.0)
+    upper_right = (pitch / 2, pitch / 2, height)
+    uniform_dist = openmc.stats.Box(lower_left, upper_right)
+    settings = openmc.Settings()
+    settings.source = openmc.IndependentSource(space=uniform_dist)
+    settings.batches = 200
+    settings.inactive = 40
+    settings.particles = 20000
+
+    if shannon_entropy:
+        entropy_mesh = openmc.RegularMesh()
+        entropy_mesh.lower_left = lower_left
+        entropy_mesh.upper_right = upper_right
+        entropy_mesh.dimension = (10, 10, 20)
+        settings.entropy_mesh = entropy_mesh
+
+    settings.temperature = {
+        "default": 280.0 + 273.15,
+        "method": "interpolation",
+        "range": (294.0, 3000.0),
+        "tolerance": 1000.0,
+    }
+    settings.export_to_xml()
+
+
+def main():
+    ap = ArgumentParser(description="SFR Pincell Model Generator")
+    ap.add_argument(
+        "-n",
+        dest="n_axial",
+        type=int,
+        default=1,
+        help="Number of cells in the Z direction",
+    )
+    ap.add_argument(
+        "-e",
+        "--shannon_entropy",
+        action="store_true",
+        help="Add Shannon entropy mesh"
+    )
+
+
+    arguments = ap.parse_args()
+    arguments = ap.parse_args()
+
+    N = arguments.n_axial
+    shannon_entropy = arguments.shannon_entropy
+    height = geom.height
```

Comment:
You don't really use this enough to assign a new variable - just use `geom.height` where necessary